### ISSUE TITLE
Misinformation correction

### ISSUE THREAD

**Issue by Gi60s** [ChiperSoft/tape-vs-mocha#1]
Great intro to Tape vs Mocha. I've used Tape quite a bit but mostly I've used Mocha for the last year or two. Now I find myself looking at Tape again to re-evaluate whether I should use it over Mocha.

I followed the examples and in [Example 2](https://github.com/ChiperSoft/tape-vs-mocha#example-2) I found your argument slightly misleading. Common testing practice says that there should only be one assertion per test. In Mocha it is possible (but bad practice) to wrap multiple assertions within a test.

The correct way to have written `mocha/test2.js` is as follows:

``` js
var assert = require('assert');
var getOtherData = require('../index2').getOtherData;

describe('getOtherData', function () {

  describe('returns the correct information', function () {

    var data = getOtherData();

    it('A', function() {
      assert.equal(data.a, 'A');
    });

    it('B', function() {
      assert.equal(data.b, 'B');
    });

    it('C', function() {
      assert.equal(data.c, 'C');
    });

  });

})
```

This example perfectly mirrors your `tape/test2.js` file. The real argument to be won here is that it's much more concise in Tape. The argument you've made for [Example 2](https://github.com/ChiperSoft/tape-vs-mocha#example-2) if faulty due to a bad mocha practice.

I don't mean to offend, I'm just trying to strengthen your argument by improving the validity of it.

**Comment by Twipped** [comment]
Example 2 is, unfortunately, rather contrived. I wrote this entire thing in like 30 minutes and didn't have time to make more realistic examples. Realistically for the response coming out of getOtherData you'd either do a deepEqual or a subset assert, which would be a single assertion, and match the pattern you're describing.

In short, I think the bad example code led you to a false assumption about the intent of the test. You're absolutely right that if those three assertions were asserting different things, they belong in separate test cases.

Here's some example tests that more accurate describe what I was trying to demonstrate:
- https://github.com/expressjs/cookie-parser/blob/master/test/cookieParser.js#L143-L167
- https://github.com/request/request/blob/master/tests/test-basic-auth.js#L58-L73
- https://github.com/visionmedia/superagent/blob/master/test/node/user-agent.js#L23-L32

I haven't touched this repo since I made it, I should probably circle back around, give it some proper examples, and turn it into a proper blog post.
